### ISSUE TITLE
New update crashes the App after clicking Install Update

### ISSUE THREAD

**Issue by Prathapkarumanchi** [BlupGithub/Blup-IDE#3]
<img width="561" alt="Screenshot 2023-07-26 at 5 30 08 PM" src="https://github.com/BlupGithub/Blup_IDE/assets/53927098/7af94038-b166-4fac-a7db-8a09b9e6280d">

**Comment by Prathapkarumanchi** [comment]
I think it's something wrong with pushing the wrong update, check the 
<img width="483" alt="Screenshot 2023-07-26 at 6 47 28 PM" src="https://github.com/BlupGithub/Blup_IDE/assets/53927098/694fb9c5-b7ed-4eb3-b466-27e77cfbeddb">
version from screenshot

**Comment by Prathapkarumanchi** [comment]
By the way no crash error in console
....
.....
```
flutter: check process 0  
flutter: check process 0  

Saving session...completed.
Deleting expired sessions...none found.

[Process completed]
```

**Comment by ashutosh2014** [comment]
Hi @Prathapkarumanchi, Thanks for reporting this bug, The bug is fixed in v1.07.
